packages/colony-starter-angular/src/app/app.component.ts

```diff
@@ -18,65 +12,96 @@ export class AppComponent implements OnInit {
   public title = "colony-starter-angular";
 
   public state = {
+    colonyClient: [],     // colonyClient (per account)
+    networkClient: [],    // networkClient (per account)
+  };
+
+  public model = {
     colony: {
       address: "",
       id: null,
     },
-    colonyClient: [],     // colonyClient (per account)
     domain: {
       id: null,
       parentSkillId: null,
       potId: null,
     },
-    networkClient: [],    // networkClient (per account)
-    task: {},
-    tokenAddress: "",
+    networkClient: {
+      accountAddr: "",
+      addr: "",
+    },
+    parentDomainId: 1,
+    task: {
+      desc: "",
+      title: "",
+    },
+    taskObject: null,
+    token: {
+      address: "",
+      name: "",
+      symbol: "",
+    },
   };
 
+  constructor(@Inject(AppService) private appService: AppService) {}
+
   public ngOnInit(): void {
-    this.example();
+    this.start();
   }
 
-  public async example() {
-
+  public async start() {
     // Create an instance of ColonyNetworkClient using the adapter
-    this.state.networkClient[0] = await connectNetwork(0);
+    this.appService.connectNetwork().then( (res) => {
+      console.log(res);
```

Comment:
Remove console.log. Maybe you forgot to remove after testing.

Reply:
Removed